src/verify.c

```diff
@@ -774,11 +774,21 @@ int verify_main(int argc, char **argv)
 		if ((file_not_fixed_count == 0) && (file_not_replaced_count == 0)) {
 			remove_orphaned_files(official_manifest);
 		}
+		if (cmdline_option_picky) {
+			char *start = mk_full_filename(path_prefix, "/usr");
+			fprintf(stderr, "--picky removing extra files under %s\n", start);
+			ret = walk_tree(official_manifest, start, true);
+			if (ret >= 0) {
+				file_checked_count = ret;
```

Comment:
I like seeing the progress output being paralleled between `--picky` and `--fix --picky`, so I think this change is appropriate.